salt/states/win_system.py

```diff
@@ -115,7 +115,7 @@ def computer_name(name):
     if result is not False:
         after_name = result['Computer Name']['Current']
         after_pending = result['Computer Name'].get('Pending')
-        if ((after_pending is not None and after_pending == name) or
+        if ((after_pending is not None and after_pending == name.upper()) or
                 (after_pending is None and after_name == name)):
```

Comment:
@cachedout I want the computer name to actually be the case the user passed and not capitalize it. The `system.get_pending_computer_name` function always returns a capitalized host name, so you must capitalize `name` when you make the comparison. I could not find another way to get a not-all-caps pending name in windows.